services/src/main/java/org/fao/geonet/api/registries/CrsUtils.java

```diff
@@ -167,7 +167,7 @@ public static Crs getById(String crsId) {
      */
     static protected boolean matchesFilter(String input, String[] filters) {
         for (String match : filters) {
-            if (!input.contains(match))
+            if (!input.contains(match.toUpperCase()))
```

Comment:
`input` should be `toUpperCase()` to then
Usually, we tranform both strings to lower case for comparison.

Reply:
It is already see https://github.com/geonetwork/core-geonetwork/blob/master/services/src/main/java/org/fao/geonet/api/registries/CrsUtils.java#L97